templates/ts-react-apollo-client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
This will mean users having this when a template is downloaded. Do we really want that?

Reply:
i.e are the problems we were seeing e.g `eslint` crashing with react-scripts' `eslint` only related to how the templates directory is setup in our repository?

Reply:
I do not know. It is just that released version of the graphql-cli doesn't work with instruction in the readme. I have tested it twice with different template options. Always needed to skip_preflight

Reply:
If you guys testing graphql-cli and **do not see this** then maybe I have react installed globally or something

Reply:
I have feeling that this might be issue on my side - you can actually check that during verification since we have separate workspace now

Reply:
I created a new project with graphql-cli fresh (outside repos) and the client starts fine without this env - are you seeing it outside repos like this?

Reply:
I do not remember having the issue when the template was downloaded outside graphback repo. In my opinion, we can remove this file and do @craicoverflow suggestion in https://github.com/aerogear/graphback/pull/1997#discussion_r483677834. It's easier to forget that so let's add it to https://github.com/aerogear/graphback/blob/master/CONTRIBUTING.md

Reply:
Ok.. I tracked the problem. I have created sample project in the folder that had number of package.json files above and some of those had react. After moving out all works.